package/ios/Core/Extensions/AVCaptureConnection+orientation.swift

```diff
@@ -9,27 +9,24 @@ import AVFoundation
 import Foundation
 
 extension AVCaptureConnection {
-  /**
-   Gets or sets the target orientation of the video connection.
-   */
-  var orientation: Orientation {
-    get {
-      if #available(iOS 17.0, *) {
-        return Orientation(degrees: videoRotationAngle)
-      } else {
-        return Orientation(videoOrientation: videoOrientation)
-      }
+  func getOrientation(device: AVCaptureDevice) -> Orientation {
```

Comment:
As I said before; I'd prefer a property (getter + setter) and remove the `device` logic from here. I don't think it belongs in here. All `sensorOrientation` logic should be handled before this.